copts.bzl

```diff
@@ -58,6 +58,7 @@ CURVE_GCC_FLAGS = [
     "-Wwrite-strings",
     "-Werror",
     "-DNOMINMAX",
+#    "-faligned-new",
```

Comment:
Delete it if don't need?

Reply:
> Delete it if don't need?

it is required in higher versions of gcc